data/projects.json

```diff
@@ -388,5 +388,15 @@
         "level": "intermediate",
         "site": "https://github.com/lmponcio/copia-backups",
         "tags": "python, copia-backups, scripting"
+    },
+    {
+        "name": "tcp server and client in python",
```

Comment:
I couldn't find this repo on your profile

Reply:
Hello @ming-tsai,
it's a github gist.

Reply:
Sorry this repo for know is only allowed the repository, I will be find out the wait to save the gists